src/client/renderingengine.cpp

```diff
@@ -485,15 +485,23 @@ void RenderingEngine::_draw_load_screen(const std::wstring &text,
 			guienv->addStaticText(text.c_str(), textrect, false, false);
 	guitext->setTextAlignment(gui::EGUIA_CENTER, gui::EGUIA_UPPERLEFT);
 
+	get_video_driver()->beginScene(true, true, video::SColor(255, 140, 186, 250));
+
 	bool cloud_menu_background = clouds && g_settings->getBool("menu_clouds");
 	if (cloud_menu_background) {
 		g_menuclouds->step(dtime * 3);
 		g_menuclouds->render();
-		get_video_driver()->beginScene(
-				true, true, video::SColor(255, 140, 186, 250));
 		g_menucloudsmgr->drawAll();
-	} else
-		get_video_driver()->beginScene(true, true, video::SColor(255, 0, 0, 0));
+	}
+
+	video::ITexture *texture = tsrc->getTexture("loadscreen_bg.png");
+	if (texture) {
```

Comment:
When I don't provide the loadscreen_bg texture I get this error:
`ERROR[Main]: generateImage(): Could not load image "loadscreen_bg.png" while building texture; Creating a dummy image`
getTexture() seems to return a pointer to an auto generated image instead of a null pointer when the texture isn't found. That texture is then displayed instead of clouds/blue background.